ui/src/store/modules/project.ts

```diff
@@ -373,6 +380,13 @@ const mutations: MutationTree<ProjectState> = {
     state.jobCollection = collection ? serializeJobCollection(collection) : null
   },
 
+  storeJob (state, job) {
+    if (!state.jobCollection) throw new Error('JobCollection not loaded')
+
+    state.jobCollection.member = state.jobCollection.member
+      .map((storedJob) => storedJob.id.equals(job.id) ? Object.freeze(job) : storedJob)
```

Comment:
`state.jobCollection.member` gets replaced when storing job? I expected a new field in the state

Reply:
Now that you mention it, I don't even need to store it in the global state. Will change this.